contact-tracing.postman_collection.json

```diff
@@ -992,7 +998,10 @@
 									"  pm.expect(tv4.validate(json, schema)).to.be.true",
 									"})",
 									"",
-									"pm.environment.set('code', json.code.toString())"
```

Comment:
The existing test script was important as it saves the values required for the APHL supporting endpoints - could you revert please?

Reply:
It looks like that check didn't exist in the 'Notify Positive' call. As that and this both return the exact same response I've added it in both.

Reply:
The `Notify positive` request (part of the old upload process) needs to generate the `uploadHash` to pass to `/exposures/verify`. However, the `Issue code` (part of the APHL supporting process) just needs to pass along the `code`  to `/verify`. The only changes that need to happen to the test scripts are the addition of `smsSent` to the response schema.

Reply:
I'm not sure I follow. They both execute the same code and return the same response. Shouldn't we be verifying all the expected values are returned?